invenio_config_tugraz/templates/security/email/welcome.txt

```diff
@@ -0,0 +1,17 @@
+{% trans salut=current_userprofile.full_name or user.email %}
+Dear {{ salut }}!
+{% endtrans %}
+{{ _('Welcome to TU Graz Repository!') }}
+{{ _('To help you get started, here are some useful links:') }}
+
+  {{ config.SITE_HOSTNAME }}{{ url_for('invenio_theme_tugraz.index') }}
+  {{ config.SITE_HOSTNAME }}{{ url_for('invenio_theme_tugraz.guide') }}
+  {{ config.SITE_HOSTNAME }}{{ url_for('invenio_theme_tugraz.terms') }}
```

Comment:
we are assigning these values - with env so the variable you are looking for is ```INVENIO_SITE_HOSTNAME```. can you test if you add the value for ```INVENIO_SITE_HOSTNAME``` into a ```.env``` file, would that be picked?

Reply:
INVENIO_SITE_HOSTNAME in .env file will be available as config.SITE_HOSTNAME - same with other variables prefixed with INVENIO_